src/components/books/BookItem.vue

```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+a(href='#')
+  .book-item(:class='{ "book-item-small": small }')
+    img.book-item-image(alt='book image' src='' v-bind:src='book.image_url || "assets/book_logo.png"')
+    p.book-item-title(v-if='!small')
```

Comment:
maybe you can wrap this into something else to do the v-if condition instead of repeating it